_posts/2023-10-11-14.0.0-release.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Apache Arrow 14.0.0 Release"
+date: "2023-10-11 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 14.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 13.0.0 release, Metehan Yildirim and Oleks V. have been invited to be committers.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+## C# notes
+
+## Go notes
```

Comment:
```suggestion
## Go Notes

* The minimum version of Go officially supported is now `go1.19` instead of `go1.17` ([GH-37636](https://github.com/apache/arrow/issues/37636))

### Bug Fixes

#### Arrow

* Documentation fixed to correctly state that the default unit for `TimestampType` is seconds ([GH-35770](https://github.com/apache/arrow/issues/35770))
* Fixed leak in the `Concatenate` function if there is a panic that is recovered ([GH-36850](https://github.com/apache/arrow/issues/36850))
* Ensure Binary dictionary indices are released on panic ([GH-36858](https://github.com/apache/arrow/issues/36858))
* Fix overflow value causing invalid dates for `MarshalJSON` on some timestamps ([GH-36935](https://github.com/apache/arrow/issues/36935))
* Fix leaking dictionary allocations in IPC reader ([GH-36981](https://github.com/apache/arrow/issues/36981))

#### Parquet

* Fixed an issue where DeltaLengthByteArray encoding fails on certain null value scenarios ([GH-36318](https://github.com/apache/arrow/issues/36318))
* Correctly propagate internal `writer.sink.Close()` errors from `writer.Close()` when writing a Parquet file ([GH-36645](https://github.com/apache/arrow/issues/36645))
* Fixed a panic when writing some specific DeltaBitPacked datasets ([GH-37102](https://github.com/apache/arrow/issues/37102))
* Proper support for Decimal256 data type in Parquet lib ([GH-37419](https://github.com/apache/arrow/issues/37419))
* Corrected inconsistent behavior in `pqarrow` column chunk reader ([GH-37845](https://github.com/apache/arrow/issues/37845))
* Rewrote and Fixed ARM64 assembly for bitmap bit extractions and integer packing ([GH-37712](https://github.com/apache/arrow/issues/37712))

### Enhancements

* C Data Interface integration testing has been added and implemented ([GH-37789](https://github.com/apache/arrow/issues/37789))
* pkg.go.dev link is fixed in the Readme ([GH-37779](https://github.com/apache/arrow/issues/37779))

#### Arrow

* Added `String()` method to `arrow.Table` ([GH-35296](https://github.com/apache/arrow/issues/35296))
* Add proper `array.Null` type support handling for arrow/csv writing ([GH-36623](https://github.com/apache/arrow/issues/36623))
* Optimized `GetOrInsert` function for memo table handling of dictionary builders ([GH-36671](https://github.com/apache/arrow/issues/36671))
* Made it possible to add custom functions in the `compute` package ([GH-36936](https://github.com/apache/arrow/issues/36936))
* Improved performance of dictionary unifier ([GH-37306](https://github.com/apache/arrow/issues/37306))
* Added direct access to dictionary builder indices ([GH-37416](https://github.com/apache/arrow/issues/37416))
* Added ability to read back values from Boolean builders ([GH-37465](https://github.com/apache/arrow/issues/37465))
* Add `ValueLen` function to string array ([GH-37584](https://github.com/apache/arrow/issues/37584))
* Avoid unnecessary copying in the default go allocator ([GH-37687](https://github.com/apache/arrow/issues/37687))
* Add `SetNull(i int)` to array builders ([GH-37694](https://github.com/apache/arrow/issues/37694))

#### Parquet

* Parquet metadata is allowed to write metadata after writing rowgroups using `pqarrow.FileWriter` ([GH-35775](https://github.com/apache/arrow/issues/35775))
* `MapOf` and `ListOf` helper functions have been improved to provide clearer error messages and have better documentation ([GH-36696](https://github.com/apache/arrow/issues/36696))
* Struct tag of `parquet:"-"` will be allowed to skip fields when converting a struct to a parquet schema ([GH-36793](https://github.com/apache/arrow/issues/36793))

```